pkg/controller/status/status.go

```diff
@@ -407,6 +416,15 @@ func (m *statusManager) syncState() bool {
 	return m.explicitDegradedReason != ""
 }
 
+// crCleanup removes the status displayed for CR in tigerastatus
```

Comment:
Nit "in the TigeraSTatuses" (I think we normally use the capitalise / camel case for resource names)